File: app/src/main/java/com/bitcoin/merchant/app/screens/legal/TermsOfUseFragment.java
package com.bitcoin.merchant.app.screens.legal;

import android.webkit.WebView;

import com.bitcoin.merchant.app.R;

public class TermsOfUseFragment extends LegalTextFragment {
    protected void setupWebView(WebView webView) {
        setToolbarTitle(R.string.menu_terms_of_use);
        webView.loadUrl("https://www.bitcoin.com/legal/");
    }
}

File: app/src/main/java/com/bitcoin/merchant/app/screens/legal/TermsOfUseFragment.kt
package com.bitcoin.merchant.app.screens.legal

import android.webkit.WebView
import com.bitcoin.merchant.app.R

class TermsOfUseFragment : LegalTextFragment() {
    override fun setupWebView(webView: WebView) {
        setToolbarTitle(R.string.menu_terms_of_use)
        webView.loadUrl(resources.getString(R.string.url_terms_of_use))
    }
}